### ISSUE TITLE
Can shardingsphere 5.0 beta version support joining tables from different dbs

### ISSUE THREAD

**Issue by number68** [apache/shardingsphere#12176]
## Question

Can shardingsphere 5.0.0 beta version support joining tables per from different db?

I found an exciting new feature in shardingsphere 5.0.0 beta version release note , details in below pic:
![image](https://user-images.githubusercontent.com/23182143/131814073-cce729fe-d319-4fb9-9cb6-b210585c9ee1.png)

But i can't follow what it exactly means.

Can shardingsphere 5.0.0 beta version support joining tables per from different db?

For example, a from db1，b from db2，maybe a,b have sharding rules(not important here).
select a.name, b.name
from a,b 
where a.bid = b.id
and a.name like "%xxx%";

**Comment by strongduanmu** [comment]
@number68 Yes, the beta version has begun to support cross-database query, this is a new feature, there may be some problems, but you can try it.

**Comment by number68** [comment]
> @number68 Yes, the beta version has begun to support cross-database query, this is a new feature, there may be some problems, but you can try it.

@strongduanmu thanks for reply, it's a great feature. 
How to deal with huge pre filting results , i.e. a filtered millon records and cross join b's filtered result and sort , do you used streaming process tool(like mpp spark)?  
What the join code classes  are?
thanks a lot!

**Comment by strongduanmu** [comment]
@number68 You can find the entrance through `FederateExecutor` and realize cross-database query with the help of Calcite. At present, the feature is still being optimized. Welcome to participate.

**Comment by number68** [comment]
> @number68 You can find the entrance through `FederateExecutor` and realize cross-database query with the help of Calcite. At present, the feature is still being optimized. Welcome to participate.

ok  thx a lot!
what config needs to be done for using this awesome feature?  @strongduanmu

**Comment by kimmking** [comment]
It's a big issue. I will try it and feedback later in another thread.
